datafusion/core/Cargo.toml

```diff
@@ -102,7 +102,6 @@ bigdecimal = "0.4.1"
 criterion = { version = "0.5", features = ["async_tokio"] }
 csv = "1.1.6"
 ctor = "0.2.0"
-datafusion-sqllogictest = { path = "../sqllogictest", version = "29.0.0", features = ["postgres"] }
```

Comment:
Removing this line to remove the circular dependency is the entire reason for this PR